src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -123,9 +123,10 @@ def daos_server_storage_prepare_reset(self, step):
             self.fail(
                 "#({0}.A){1} failed, "
                 "please make sure the server equipped with PMem modules".format(step, cmd))
+        time.sleep(5)
```

Comment:
Is there a more deterministic way to determine the test can continue here?  We should avoid arbitrary sleeps in test code.  

Also checking `results[0]['exit_status']` is ill-advised as there could be multiple results (max of one per self.hostlist_servers) even if right now the test only uses one server host - as launch.py can easily override this.

Also using 'run_remote' in place of 'run_pcmd' would make checking the results easier as well as defining a DaosServerCommand.scm_seset() helper function.